### PR TITLE
Avoid erroring when OMP_NUM_THREADS is empty string

### PR DESCRIPTION
**Pull request by phofl**
unsetting the variable like this is not uncommon but this will raise here unfortunately

This circumvents the issue

### CODE REVIEW

File: numexpr/utils.py [hunk 1]
```diff
@@ -152,7 +152,8 @@ def _init_num_threads():
     # actual number of threads used.
     if 'NUMEXPR_NUM_THREADS' in os.environ:
```
Comment (bashtage) [inline numexpr/utils.py:153]: Would be good to fix this one as well, and any others that blindly call int on an environmental variable.  Would it be more pythonic to try/except and then reraise (or warn) with an explanation?

Comment (phofl) [reply numexpr/utils.py:153]: I think an empty string should be fine, so just ignoring it seems good to me

**Comment by phofl** [comment]
Updated

**Comment by FrancescAlted** [comment]
Unfortunately test suite is not passing with the latest commit.

**Comment by phofl** [comment]
should be fixed now

**Comment by FrancescAlted** [comment]
Looks good.  Thanks @phofl !

**Pull request merged**